### PR TITLE
UCT/UCS/MEM: Limit allocation size for huge pages.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5307/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask pls take a look

File: src/uct/sm/mm/mm_sysv.c [hunk 1]
```diff
@@ -46,8 +46,8 @@ uct_sysv_alloc(uct_md_h md, size_t *length_p, ucs_ternary_value_t hugetlb,
     }
 
     if (hugetlb != UCS_NO) {
-        status = ucs_sysv_alloc(length_p, address_p, flags | SHM_HUGETLB, &shmid
-                                UCS_MEMTRACK_VAL);
+        status = ucs_sysv_alloc(length_p, (*length_p) * 2, address_p,
```
Comment (alinask) [inline src/uct/sm/mm/mm_sysv.c]: why *2?

Comment (yosefe) [reply src/uct/sm/mm/mm_sysv.c]: setting the maximal size an allocation can be expanded as 2x

Comment (alinask) [reply src/uct/sm/mm/mm_sysv.c]: yes but why 2?

**Pull request merged**